netplugin/cluster/cluster.go

```diff
@@ -178,7 +214,11 @@ func MasterPostReq(path string, req interface{}, resp interface{}) error {
 
 		log.Infof("Making REST request to url: %s", url)
 
-		err := HTTPPost(url, req, resp)
+		if isDel {
```

Comment:
Here we are trying to reach the leader netmaster with retry. If it not accessible then we try to reach other netmasters. Not sure if we need to add more retries here as we are already trying multiple netmasters.